vendor/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.pb.go

```diff
@@ -17574,10 +17574,7 @@ func (m *VersionRequest) Unmarshal(dAtA []byte) error {
 			if err != nil {
 				return err
 			}
-			if skippy < 0 {
-				return ErrInvalidLengthApi
-			}
-			if (iNdEx + skippy) < 0 {
+			if (skippy < 0) || (iNdEx+skippy) < 0 {
```

Comment:
I see k8s 1.21 has the updated protos with the gogo/protobuf fixes for  [CVE-2021-3121](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-3121) (similar to https://github.com/containerd/containerd/pull/4974)

Reply:
Ah; looks like it was backported to 1.20 (https://github.com/kubernetes/kubernetes/pull/100501), but not yet in 1.20.5 (https://github.com/kubernetes/kubernetes/blob/v1.20.5/go.mod#L53)

@dims do you know when 1.20.6 is planned? https://github.com/kubernetes/kubernetes/compare/v1.20.5...release-1.20